Volume_1/Book_about_Quadratization.tex

```diff
@@ -1,7 +1,7 @@
 %% LyX 2.2.0 created this file.  For more info, see http://www.lyx.org/.
 %% Do not edit unless you really know what you are doing.
 %\documentclass[english,notitlepage,longbibliography,showpacs,preprintnumbers,amsmath,amssymb,aps,prl,nofootinbib,12pt,superscriptaddress]{revtex4-1}
-\documentclass[english,notitlepage,longbibliography,showpacs,preprintnumbers,amsmath,amssymb,aps,prx,nofootinbib,12pt,superscriptaddress]{revtex4-1}
+\documentclass[english,notitlepage,longbibliography,showpacs,preprintnumbers,amsmath,amssymb,aps,prx,nofootinbib,12pt,superscriptaddress]{revtex4-2}

```

Comment:
@ehuan2  Why did you have to switch to RevTeX 4.2?

Reply:
yeah it's weird, I think it didn't compile before locally with 1, but I can revert it (worked online with overleaf)